.github/workflows/tests-coverage.yml

```diff
@@ -31,10 +31,14 @@ jobs:
             target/
           key: ${{ runner.os }}-cargo-${{ hashFiles('Cargo.lock') }}
           fail-on-cache-miss: true
-      - name: Run test
+      - name: Run test (native)
         run: |-
           cd tests
           npm run test
+      - name: Run test (wasm)
+        run: |-
+          cd tests
+          FORCE_WASM_EXECUTION=true npm run test
```

Comment:
You were able to confirm the right execution environment was executed?
You can be sure by putting a `print` in the pallet, behind a `sp_std::if_std` macro. It will be printed to stdout in native but not in wasm